web/src/stream_pill.ts

```diff
@@ -22,6 +22,7 @@ export function display_pill(sub: StreamSubscription): string {
 export function create_item_from_stream_name(
     stream_name: string,
     current_items: CombinedPillItem[],
+    display_pill_func?: (sub: StreamSubscription) => string,
```

Comment:
not sure about the parameter name.

Reply:
Because of this line:
```
display_value: display_pill_func ? display_pill_func(sub) : display_pill(sub),
````
If we change the parameter name to `display_pill` then the code will always refer to the parameter even when we want to call the `stream_pill.display_pill`.

Reply:
Ah, the name seems fine in that case.

Reply:
My feedback on naming elsewhere applies, but one extra note is that we don't use abbreviations like "_func" as a project policy. A good name shouldn't require such a suffix anyway; something like `stream_to_pill_label` is going to be much better in most cases anyway.

Reply:
Okay, I noticed `update_func` in `opts` within `pill_typeahead.ts` and thought we could utilize that suffix. However, that seems to be a rare use case for suffixes, I guess.